pom.xml

```diff
@@ -198,16 +198,104 @@
                         <repositoryOwner>OpenTOSCA</repositoryOwner>
                         <repositoryName>mvn-repo</repositoryName>
                     </configuration>
-                    <!--
+                </plugin>
+                <plugin>
+                    <groupId>org.reficio</groupId>
+                    <artifactId>p2-maven-plugin</artifactId>
+                    <version>1.3.0</version>
                     <executions>
                         <execution>
-                            <goals>
-                                <goal>site</goal>
-                            </goals>
-                            <phase>site</phase>
+                            <id>default-cli</id>
+                            <configuration>
+                                <artifacts>
+                                    <artifact>
+                                        <id>org.eclipse.winery:org.eclipse.winery.model.bpmn4tosca:2.0.0-SNAPSHOT</id>
```

Comment:
We really need to give specific versions here? So at each release, we need to update this file?

Reply:
Yes, according to the docs it seems so.